src/parameterizations/vertical/MOM_diabatic_driver.F90

```diff
@@ -287,8 +290,25 @@ subroutine diabatic(u, v, h, tv, Hml, fluxes, visc, ADp, CDp, dt, Time_end, &
   integer :: i, j, k, m, is, ie, js, je, nz
   logical :: showCallTree ! If true, show the call tree
 
+  real, dimension(SZI_(G),SZJ_(G),SZK_(G))   :: h_in  ! thickness before thermodynamics
```

Comment:
The allocations need to be behind a flag controlling whether ocn stoch is active.

Reply:
will take care of this